src/Propel/Generator/Reverse/PgsqlSchemaParser.php

```diff
@@ -417,7 +417,10 @@ protected function addForeignKeys(Table $table, $oid)
                 $fk = new ForeignKey($name);
                 $fk->setForeignTableCommonName($foreignTable->getCommonName());
                 if ($table->guessSchemaName() != $foreignTable->guessSchemaName()) {
-                    $fk->setForeignSchemaName($foreignTable->getSchema());
+                     if($foreignTable->getSchema())
+                            $fk->setForeignSchemaName($foreignTable->getSchema());
+                     elseif($foreignTable->getSchema() == 'public')
```

Comment:
Hi Marc, sorry you were right, i uploaded the wrong commit, please have a look at it now